src/EditorFeatures/Test/CommentSelection/AbstractToggleCommentTestBase.cs

```diff
@@ -84,11 +86,12 @@ private static void SetupSelection(IWpfTextView textView, IEnumerable<Span> span
         }
 
         [ExportWorkspaceService(typeof(IExperimentationService), WorkspaceKind.Test), Shared]
-        private class MockToggleBlockCommentExperimentationService : IExperimentationService
+        private class MockToggleCommentExperimentationService : IExperimentationService
```

Comment:
I think this could be a generic service thing we add to base test MEF exporter if  you want to.

Reply:
I don't think it's unlikely that other tests will need to override experiments differently (e.g. test experiment on and off)